### PR TITLE
W 12213767 add policy page

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/manage-policies-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+= Gateway Policies in API Mananager
```
Comment (sathya0) [inline modules/ROOT/pages/manage-policies-overview.adoc]:
```suggestion
= Gateway Policies in API Manager
```

File: modules/ROOT/pages/manage-policies-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:page-aliases: 
+
+From API Mananager, you can apply policies to APIs in Flex Gateway Connected Mode or in Mule Gateway.
```
Comment (sathya0) [inline modules/ROOT/pages/manage-policies-overview.adoc]:
```suggestion
From API Manager, you can apply policies to APIs in Flex Gateway Connected Mode or in Mule Gateway.
```

File: modules/ROOT/pages/manage-policies-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+* xref:policies::policies-included-directory.adoc[Included Policies Directory]
+
+== Applying Policies in Flex Gateway Connected Mode and in Mule Gateway
+
+* xref:policies::policies-included-apply.adoc[Apply an Included Policy]
+
+== See Also
+
+* xref:policies::policies-included-reorder.adoc[]
```
Comment (sathya0) [inline modules/ROOT/pages/manage-policies-overview.adoc:26]: just checking if you wanted to include a title in these brackets

Comment (glenn-rodgers-sf) [reply modules/ROOT/pages/manage-policies-overview.adoc:26]: Yeah, it just renders as the page title

**Review by sathya0 — APPROVED** [review]

**Pull request merged**